src/test/java/uk/gov/homeoffice/digital/sas/person/listeners/PersonKafkaEntityListenerTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.homeoffice.digital.sas.person.listeners;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.homeoffice.digital.sas.kafka.producer.KafkaProducerService;
+import uk.gov.homeoffice.digital.sas.person.enums.TermsAndConditions;
+import uk.gov.homeoffice.digital.sas.person.model.Person;
+
+@ExtendWith(MockitoExtension.class)
+class PersonKafkaEntityListenerTest {
+
+  private final static String TENANT_ID = "b7e813a2-bb28-11ec-8422-0242ac120002";
+  private final static String PERSON_ID = "7f000001-867e-1c1f-8186-7edc2eb90000";
+
+  private Person person;
+
+  @Mock
+  private KafkaProducerService<Person> kafkaProducerService;
+
+  private final PersonKafkaEntityListener personKafkaEntityListener =
+      new PersonKafkaEntityListener(kafkaProducerService);
+
+  @BeforeEach
+  void setup() {
+    person = Person.builder()
+        .firstName("John")
+        .lastName("Smith")
+        .version(1)
+        .fteValue(new BigDecimal("1.0000"))
+        .termsAndConditions(TermsAndConditions.MODERNISED)
+        .build();
```

Comment:
nit: you could probs get away with not having to set any of these fields.
No harm but doesn't seem relevant to the test.

Reply:
The only field that can be left out, without failing the test, is fteValue. All the others are mandatory (@NotNull).
Still, I feel it's sensible to include it for completeness

Reply:
`@NotNull` would only take effect when invoking the validator. Here it's just a plain instantiation so should still pass.

My thinking leans towards keeping test setup to a minimum for what's required to make the test fulfill it's scenario and pass.
But that's only my personal preference and absolutely no harm to include. Will resolve 👍